supply_energy.md

```diff
@@ -0,0 +1,291 @@
+| :--- | :--- | :--- | :--- |
+|  |  |  | [Exogenous](inputs_supply.html) |
+
+
+## Description
+
+### Resources
+
+#### Depletable Resources 
+
+GCAM models depletable resources (oil, unconventional oil, natural gas, coal, and uranium) using graded resource supply curves. The fossil resources are produced from these supply curves using a "Resource / Reserve" model.  In this approach as the market price of the resource increases, we look up the [supply curve](details_energy.html#depletable-resources) to determine the additional quantity available and move that quantity of "resource" into a "reserve".  We assume production of that reserve over some well / mine lifetime appropriate for each fuel.  Technical change can be applied to reduce the extraction cost of the "resource" in future years. Resource curves for fossil fuels are shown below.
+
+#### Renewable Resources 
+
+GCAM's renewable resources include wind, solar, geothermal, hydropower, and biomass; some regions are also assigned a "traditional biomass" resource. In contrast to the depletable resources, whose cumulative stocks are explicitly tracked, renewable resource quantities in GCAM are always indicated in terms of annual flows. Wind, solar, geothermal, and hydropower are considered only as options for producing electricity, and are not traded between regions. Traditional biomass is only used by the buildings sector in selected regions. Of the six renewable energy resources, only biomass is (a) traded globally, and (b) used as an energy form or feedstock in a wide variety of sectors.
+
+In general, the costs of producing electricity from renewable energy forms consist of the sum of the resource costs described here, the technology costs, and in some cases, backup-related costs. The latter two components to the costs are documented in the [electricity sector](supply_energy.html#electricity).
+
+##### Wind
+
+All regions are assigned a wind energy supply curve, where the quantity is in exajoules (EJ) of electricity produced per year, and the price is in 1975$ per gigajoule (GJ) of electricity produced. Unlike fossil resources, uranium, or biomass, the quantities of wind energy are considered to be within "regional" markets; they can not be traded between the different modeled geopolitical regions. The [supply curves](details_energy.html#wind) in each region are derived from bottom-up analysis documented in [Zhou et al. (2012)](supply_energy.html#zhou2012). Note that in this supply curve formulation, the price is zero when the quantity is zero; this is because the wind resource supply curves only represent costs that increase with deployment due to the nature of the resources; these include factors such as increased transmission distances, reduced capacity factors, and more costly access to sites, among others. The remainder of the costs of wind electricity generation are in the technology and backup, described in the [electricity sector](supply_energy.html#electricity).
```

Comment:
Suggested revision to reflect new onshore wind data set and inclusion of offshore wind.  

"All regions are assigned onshore wind and offshore wind energy supply curves, where the quantity is in exajoules (EJ) of electricity produced per year, and the price is in 1975$ per gigajoule (GJ) of electricity produced. Unlike fossil resources, uranium, or biomass, the quantities of wind energy are considered to be within "regional" markets; they can not be traded between the different modeled geopolitical regions. The [supply curves](details_energy.html#wind) in each region are derived from bottom-up analysis documented in [Eurek et al. (2017)](supply_energy.html#eurek2017). Note that in this supply curve formulation, the price is zero when the quantity is zero; this is because the wind resource supply curves only represent costs that increase with deployment due to the nature of the resources; these include factors such as reduced capacity factors, and water depth for offshore wind. Region-specific grid connection costs are derived from data on wind resource distance to grid, also from [Eurek et al. (2017)](supply_energy.html#eurek2017). The remainder of the costs of wind electricity generation are in the technology and backup, described in the [electricity sector](supply_energy.html#electricity).